client/extensions/woocommerce/app/dashboard/widgets/stats-widget/stat.js

```diff
@@ -0,0 +1,117 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { findIndex } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Delta from 'woocommerce/components/delta';
+import formatCurrency from 'lib/format-currency';
+import { isRequestingSiteStatsForQuery } from 'state/stats/lists/selectors';
+import Sparkline from 'woocommerce/components/d3/sparkline';
+import StatsModulePlaceholder from 'my-sites/stats/stats-module/placeholder';
+
+class StatsWidgetStat extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			id: PropTypes.number,
+			slug: PropTypes.string,
+		} ),
+		label: PropTypes.string.isRequired,
+		attribute: PropTypes.string.isRequired,
+		type: PropTypes.string.isRequired,
+		data: PropTypes.array.isRequired,
+		date: PropTypes.string.isRequired,
+		delta: PropTypes.object.isRequired,
+	};
+
+	renderDelta = () => {
+		const { delta } = this.props;
+		if ( ! delta || ( ! delta.classes && ! delta.direction ) ) {
+			return null;
+		}
+
+		if ( delta.classes ) {
+			return <Delta value={ delta.value } className={ delta.classes.join( ' ' ) } />;
+		}
+
+		const deltaValue =
+			delta.direction === 'is-undefined-increase'
+				? '-'
+				: Math.abs( Math.round( delta.percentage_change * 100 ) );
+
+		return (
+			<Delta
+				value={ `${ deltaValue }%` }
+				className={ `${ delta.favorable } ${ delta.direction }` }
+			/>
+		);
+	};
+
+	renderValue = ( value, data ) => {
+		const { type } = this.props;
+		switch ( type ) {
+			case 'currency':
+				return formatCurrency( value, data.currency );
+			case 'percent':
+				return value + '%';
```

Comment:
I have no idea but, do we need to have this be a translated string? Maybe in RTL this could be backwards?